docs/api/exchange_rates.rst

```diff
@@ -0,0 +1,455 @@
+                "ratio":{
+                    "errors":[
+                        "Please enter exchange rate ratio."
+                    ]
+                },
+                "sourceCurrency":{
+                    "errors":[
+                        "This value is not valid."
+                    ]
+                },
+                "targetCurrency":{
+                    "errors":[
+                        "This value is not valid."
+                    ]
+                }
+            }
+        }
+    }
+
+Getting a Single Exchange Rate
+------------------------------
+
+To retrieve the details of an exchange rate you will need to call the ``/api/v1/exchange-rates/{firstCurrencyCode}-{secondCurrencyCode}`` endpoint with the ``GET`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/exchange-rates/{firstCurrencyCode}-{secondCurrencyCode}
+
++--------------------+----------------+--------------------------------------+
+| Parameter          | Parameter type | Description                          |
++====================+================+======================================+
+| Authorization      | header         | Token received during authentication |
++--------------------+----------------+--------------------------------------+
+| firstCurrencyCode  | url attribute  | First code of currency               |
++--------------------+----------------+--------------------------------------+
+| secondCurrencyCode | url attribute  | Second code of currency              |
++--------------------+----------------+--------------------------------------+
+
+Example
+^^^^^^^
+
+To see the details for the exchange rate between ``Euro (code = EUR)`` and ``British Pound (code = GBP)`` use the below method:
+
+.. code-block:: bash
+
+     $ curl http://demo.sylius.orgg/api/v1/exchange-rates/EUR-GBP \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json"
+
+.. note::
+
+    The *EUR* and *GBP* codes are just an example.
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+     STATUS: 200 OK
+
+.. code-block:: json
+
+   {
+        "id":1,
+        "ratio":0.85157,
+        "sourceCurrency":{
+            "id":2,
+            "code":"EUR",
+            "_links":{
+                "self":{
+                    "href":"\/api\/v1\/currencies\/EUR"
+                }
+            }
+        },
+        "targetCurrency":{
+            "id":3,
+            "code":"GBP",
+            "_links":{
+                "self":{
+                    "href":"\/api\/v1\/currencies\/GBP"
+                }
+            }
+        },
+        "updatedAt":"2017-02-23T15:00:53+0100",
+        "_links":{
+            "self":{
+                "href":"\/api\/v1\/exchange-rates\/EUR-GBP"
+            }
+        }
+    }
+
+.. warning::
+
+    The order of currencies in request is not important. It doesn't matter if you will request the exchange rate for ``EUR-GBP`` or ``GBP-EUR``
```

Comment:
A link to the exchange rate's docs would be helpful to know why the order is not important.